filekit-core/src/jsMain/kotlin/io/github/vinceglb/filekit/core/FileKit.js.kt

```diff
@@ -14,6 +14,21 @@ import kotlin.coroutines.resume
 import kotlin.coroutines.resumeWithException
 import kotlin.coroutines.suspendCoroutine
 
+public fun createVisuallyHiddenInput(): HTMLInputElement {
+    val input = document.createElement("input") as HTMLInputElement
+    input.style.setProperty("position", "absolute")
+    input.style.setProperty("width", "1px")
+    input.style.setProperty("height", "1px")
+    input.style.setProperty("border", "0")
+    input.style.setProperty("padding", "0")
+    input.style.setProperty("margin", "-1px")
+    input.style.setProperty("white-space", "nowrap")
+    input.style.setProperty("clip-path", "inset(100%)")
+    input.style.setProperty("clip", "rect(0, 0, 0, 0)")
+    input.style.setProperty("overflow", "hidden")
```

Comment:
By looking at an existing js library for downloading files (594 000 downloads / week), they only add the input display to none. I think we can do the same. I tested and it works fine on Safari and Chrome.

https://github.com/kennethjiang/js-file-download/blob/0c20260f6f3dbbc94082f9f204e32322e7323294/file-download.js#L14

```suggestion
    input.style.display = "none"
```